addons/account/models/account_move.py

```diff
@@ -1338,56 +1338,128 @@ class AccountPartialReconcile(models.Model):
         help='Utility field to express amount currency')
     company_id = fields.Many2one('res.company', related='debit_move_id.company_id', store=True, string='Currency')
 
-    def create_exchange_rate_entry(self, aml_to_fix, amount_diff, diff_in_currency, currency, move_date):
-        """ Automatically create a journal entry to book the exchange rate difference.
-            That new journal entry is made in the company `currency_exchange_journal_id` and one of its journal
-            items is matched with the other lines to balance the full reconciliation.
+    @api.multi
+    def _prepare_exchange_diff_line_to_reconcile(self, amount_diff, currency,
```

Comment:
keep the method definitions on one line, we do **not** keep the 80 char limit ([cf guidelines](https://www.odoo.com/documentation/10.0/reference/guidelines.html#python))